lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -540,15 +580,157 @@ def test_cache_map_bad_dir_path(self, monkeypatch, tmp_path):
                 tmp_path, "dir_name"
             )
             tb.cache_map(tar_dir)
-
-            file_path = "/dir_name/subdir1/f11.txt"
-            expected_msg = f"The directory path {file_path!r} is an absolute path, we expect relative path to the root directory."
             with pytest.raises(BadDirpath) as exc:
                 tb.get_info(Path(file_path))
             assert str(exc.value) == expected_msg
 
-    def test_cache_map_traverse_get_info_cmap(self, monkeypatch, tmp_path):
-        """Test to build the cache map of the unpacked tarball"""
+    @pytest.mark.parametrize(
+        "file_path, expected_msg",
+        [
+            ("dir_name/subdir1/./f11.txt", "f11.txt"),
+            ("dir_name/subdir1//f11.txt", "f11.txt"),
+        ],
+    )
+    def test_cache_map_irregular_path(
+        self, monkeypatch, tmp_path, file_path, expected_msg
+    ):
+        """Test to check irreguraties in file/directory path"""
```

Comment:
s/irreguraties/irregularities/ ?

Also, it would be good if the function name and/or docstring mentioned that these irregularities should produce a _successful_ result.  🙂